Source/ACE/Entity/Player.cs

```diff
@@ -1605,5 +1605,30 @@ public void ClearDelayedTeleport()
             DelayedTeleportTime = DateTime.MinValue;
             WaitingForDelayedTeleport = false;
         }
+
+        override public void GameLoopUpdate()
+        {
+            uint oldHealth = Health.Current;
```

Comment:
Maybe we should start to add a interface to set the guidelines of things that tick or just add a base overidable tick to wo and call it as needed along with a isdirty flag.

I am not a fan of using tons of separate dictionary locks for each type such as creature, player, moved world objects.
 
I know this is a by much for this pr, but we should think about how this is going to work as the code base gets bigger..  Right now we have player loops, moved item loops and not creature vital loops..  seems to me like it could all be one lock of the list and each item can have the own method for what to do when it's ticked.